cau12.md

```diff
@@ -0,0 +1,8 @@
+###12. Lấy blog được tạo bởi user mà user này không có bất kỳ comment ở blog
+```mysql
+select bl.*
+from user as us inner join blog as bl on us.id=bl.user_id
```

Comment:
trong bảng blog mình có user_id rồi em ko phải join qua làm gì nữa đâu. Đứng từ bảng blog where user_id not in luôn